lib/src/emoji_picker_extended_utils.dart

```diff
@@ -0,0 +1,72 @@
+import 'dart:convert';
+import 'dart:math';
+
+import 'package:emoji_picker_flutter/emoji_picker_flutter.dart';
+import 'package:emoji_picker_flutter/src/recent_emoji.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'emoji_picker_internal_utils.dart';
+
+/// Helper class that provides extended usage
+
+class EmojiPickerExtendedUtils {
```

Comment:
I actually preferred the previous name, because only this utils would be exposed to the users and there is no need of making it unnecessary long. What do you think?

Reply:
ok, i didn't think about that